src/react-decathlon/README.md

```diff
@@ -0,0 +1,83 @@
+<p align="center">
+  <img src="https://raw.githubusercontent.com/meilisearch/integration-guides/master/assets/logos/logo.svg" alt="MeiliSearch-Vue" width="200" height="200" />
+</p>
+<h1 align="center">React + Meilisearch tutorial</h1>
+
+<h4 align="center">
+  <a href="https://github.com/meilisearch/meilisearch">Meilisearch</a> |
+  <a href="https://docs.meilisearch.com">Documentation</a> |
+  <a href="https://www.meilisearch.com">Website</a> |
+  <a href="https://blog.meilisearch.com">Blog</a> |
+  <a href="https://twitter.com/meilisearch">Twitter</a> |
+  <a href="https://docs.meilisearch.com/faq">FAQ</a>
+</h4>
+
+<p align="center">
+  <a href="https://slack.meilisearch.com"><img src="https://img.shields.io/badge/slack-meilisearch-blue.svg?logo=slack" alt="Slack"></a>
+  <a href="https://github.com/meilisearch/meilisearch/discussions" alt="Discussions"><img src="https://img.shields.io/badge/github-discussions-red" /></a>
+</p>
+
+<p align="center">How to implement instant search within 5 minutes in your React app!</p>
+
+## 📖 Documentation
+
+See our [Documentation](https://docs.meilisearch.com/learn/security/tenant_tokens.html#what-is-multitenancy) or our [API References](https://docs.meilisearch.com/reference/api/).
+
+## 🎁 Content of this repository
+In this repository, you will find a React application that leverages Meilisearch to create a search-as-you-type experience. This application is the outcome of the tutorial titled [How to implement instant search in your React app!](https://blog.meilisearch.com/how-to-implement-instant-search-within-5-minutes-in-your-react-app/).
+
+The `AppInitial.js` file contains the boilerplate code to follow along the tutorial. It has the exact same code as `App.js` but with missing blocks. 
```

Comment:
By just reading this, I don't get why one file has "missing blocks". Instinctively, I would go check the other one as it is apparently more complete! Maybe having a little explanation of the differences would be great 👍